composer.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "prestashop/php-coding-standards",
+  "description": "PrestaShop coding standard",
+  "version": "0.0.1",
+  "require": {
+    "symfony/console": "~3.2 || ~4.0",
+    "friendsofphp/php-cs-fixer": "^2.14",
+    "phpstan/phpstan": "^0.11.5",
```

Comment:
Well, that's what I was talking about.

In my opinion PHPStan has nothing to do here.

Reply:
It'll help to configure the boostrap.php & phpstan.neon which are specific to a PrestaShop configuration